envisage/extension_point_changed_event.py

```diff
@@ -27,3 +27,9 @@ def __init__(self, extension_point_id=None, **kw):
         self.extension_point_id = extension_point_id
 
         return
+
+    def __repr__(self):
+        return "ExtensionPointChangedEvent(index={!r}, removed={!r}, " \
+            "added={!r}, extension_point_id={!r})".format(
+                self.index, self.removed, self.added, self.extension_point_id
+            )
```

Comment:
Nit, from PEP 8:

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

Could we use brackets here instead of `\`?  It is generally easier to read.

Ultra-Nit: It would be nice to put `extension_point_id` first so it matches `__init__` better.